src/Module.h

```diff
@@ -43,6 +43,26 @@ enum class Output {
     stmt_html,
 };
 
+class [[deprecated("Use OutputFile instead of Output")]] Output {
```

Comment:
IMHO we should still prefer to use `HALIDE_ATTRIBUTE_DEPRECATED()` here, as it allows downstream folks to disable it via `HALIDE_ALLOW_DEPRECATED`